src/mpid/ch4/generic/am/mpidig_am_part.h

```diff
@@ -61,12 +61,8 @@ MPL_STATIC_INLINE_PREFIX int MPIDIG_post_pready(MPIR_Request * part_sreq, int is
 
     /* Send data when all partitions are ready */
     if (MPIR_cc_get(MPIDIG_PART_REQUEST(part_sreq, u.send).ready_cntr) ==
-        part_sreq->u.part.partitions) {
```

Comment:
We need to update the function header comment as well.

Reply:
Done.